app/views/shared/_sidebar.html.erb

```diff
@@ -4,15 +4,17 @@
 <ul class="nav nav-pills flex-column">
 
   <%# Overview %>
-  <% fullpath = lecture_path(lecture)%>
-  <% active_class = get_class_for_path(lecture_path()) %>
-  <li class="sidebar-item nav-link <%= active_class %>
-          <%= !lecture ? "disabled" : ""%>">
-    <a href="<%= url_for(fullpath)%>">
-      <i class="bi bi-house<%= active_class.present? ? "-fill" : "" %>"></i>
-      <%= t('basics.overview') %>
-    </a>
-  </li>
+  <% if lecture %>
```

Comment:
Technically, this check for `lecture` (and the other ones in this file) are no longer necessary because the hotfix prepended a check for `lecture` in `app/views/layouts/application.html.erb`. But maybe we want to keep them in for now until we think of a different solution.

Reply:
Ah ok, yeah, I think we can keep them for now and then replace them once we've come up with a better solution.